vNext/extensions/applicationinsights-react-native/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "inlineSources": true,
+    "noImplicitAny": false,
+    "module": "es6",
+    "moduleResolution": "Node",
+    "target": "es5",
+    "alwaysStrict": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "allowSyntheticDefaultImports": true,
+    "importHelpers": true,
+    "noEmitHelpers": true,
+    "forceConsistentCasingInFileNames": true,
+    "declaration": true,
+    "declarationMap": true,
+    "declarationDir": "vNext/extensions/applicationinsights-react-native/dist-esm",
```

Comment:
assuming there is no browser output for this, hence only one build and no rollup.

Reply:
correct. only es6 output for now